parkbenchcommon/ramdisk.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 Joel Allen Luellwitz and Emily Frost
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Provides a class for managing ramdisks."""
+
+__all__ = ['Ramdisk', 'RamdiskMountError', 'RamdiskOptionError']
+
+import logging
+import os
+import subprocess
+
+class RamdiskMountError(Exception):
+    """Raised when a ramdisk mount operation fails."""
+
+class RamdiskOptionError(ValueError):
+    """Raised when an option for a ramdisk mount is invalid."""
+
+
+class Ramdisk:
+    """A class for managing ramdisks."""
+
+    def __init__(self, path):
+        """Stores mountpoint and options.
+
+        path: A string indicating the path where the ramdisk will be mounted.
+        """
+        self.logger = logging.getLogger(__name__)
+
+        # The mount command outputs the canonical path of all mountpoints, so it will need
+        #   to know the absolute path with any links resolved. Python calls this realpath.
```

Comment:
I'd probably just say, "The mount command outputs the canonical path of all mountpoints. Python calls this realpath." I think the rest is clear.